Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:latest
+
+ENV TZ=Europe/Kiev
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install -y \
+    cmake \
+    g++ \
+    build-essential \
+    gperf \
+    openssl \
+    libssl-dev \
+    zlib1g-dev \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+
+RUN git clone --recursive https://github.com/tdlib/telegram-bot-api.git && \
+    cd telegram-bot-api/ && \
+    mkdir build && \
+    cd build
+
+WORKDIR telegram-bot-api/build
+
+RUN cmake .. -DCMAKE_BUILD_TYPE=Release \
+    && cmake --build . --target install
+
+CMD ["telegram-bot-api","--api-id=$API_ID", "--api-hash=$API_HASH", "--local"]
```

Comment:
There is no need to introduce new environment variables, see the Usage section of README.md:
> specify them using the `--api-id` and `--api-hash` options or the `TELEGRAM_API_ID` and `TELEGRAM_API_HASH` environment variables.